ci/README.md

```diff
@@ -0,0 +1,5 @@
+1. To deploy run:
```

Comment:
Needs more explanation and context around using and running this.

Reply:
Added more info to the readme file.